shells/pipes-shell-2/web/deploy/source/index.html

```diff
@@ -11,26 +11,46 @@
 <html lang="en">
 <head>
 
-  <title>Pipes (Arcs)</title>
+  <title>Arcs</title>
 
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 
-  <script type="module" src="./shell.js"></script>
-
   <style>
-    body, input, button {
-      font-family: sans-serif;
+    body {
+      margin: 0;
+      min-width: 320px;
+      overflow-x: hidden;
+      overflow-y: scroll;
+      background-color: white;
     }
-    [slotid="root"] {
-      padding: 4px;
-      border: 1px dotted gray;
+    body, button, input {
+      font-family: 'Google Sans', sans-serif;
```

Comment:
It's my expectation that 'sans-serif' will call up the correct Sans for the platform (used to be Roboto, is it Google Sans now?) ... what did you see?

Reply:
This page is not shown so I think the font doesn't matter. I removed Google Sans as sans-serif looks more universal too me.